tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -104,18 +104,21 @@ module.exports = {
      *
      * @param {string} fileName
      */
-    deleteFile: async function (fileName) {
-      await this
+    deleteFile: function (fileName) {
+      return this
         .waitForFileVisible(fileName)
         .useXpath()
         .performFileAction(fileName, 'delete')
         .waitForElementEnabled(
           this.elements.deleteFileConfirmationBtn.selector,
           'css selector'
         )
-
+        .clickDeleteButton()
+    },
+    clickDeleteButton: async function () {
```

Comment:
that function does not click the delete button but confirms the delete action
```suggestion
    confirmDeletion: async function () {
```